src/setup-docker/templates/fabric-docker/scripts/chaincode-functions-v2.sh

```diff
@@ -12,64 +12,69 @@ dockerPullIfMissing() {
 chaincodeBuild() {
```

Comment:
I would recommend to split it into two functions:
`chaincodeBuildDirPath`, `chaincodeBuildImage` It would simplify the logic chaincode-install-v2

Reply:
Ok 😊